### PR TITLE
feat: integrate nature theme based name to cli (render template and store service) for file artifacts

### PR DESCRIPTION
**Pull request by Peeeekay**
Changelog picked up from commits here:

feat: integrate nature theme named to render_template and store_service

### CODE REVIEW

**Comment by Peeeekay** [comment]
I am basically integrating the name themed auto-generation of file-artifacts name to the CLI.

Let me know if you have any questions, but I was stuck between either updating the current response from enclaveCtx methods like `UploadFiles` etc, because we only return UUIDs ( was reluctant to break the contract for SDK apis) or create new a RPC method which I can call to get a unique nature themed name.

I went with the latter as it I wanted to make this PR backward compatible as this particular feature should not be breaking imo opinion.

**Review by gbouv — COMMENTED** [review]
Not a big fan of that implementation TBH. I don't think we want to expose such a specialized and narrow functionality to our API. What about instead we make the artifactName arg option on our UploadFiles API function, and handle the mechanics of generating a unique one entirely in the backend?

File: cli/cli/commands/files/storeservice/storeservice.go [hunk 1]
```diff
@@ -35,17 +35,20 @@ const (
 	kurtosisBackendCtxKey = "kurtosis-backend"
```
Comment (gbouv) [inline cli/cli/commands/files/storeservice/storeservice.go:35]: nit: can we rename this file and package `storeservicefile` maybe?

**Review by gbouv — APPROVED** [review]

File: cli/cli/commands/files/storeservice/storeservice.go (hunk 1 above)
Comment (Peeeekay) [reply cli/cli/commands/files/storeservice/storeservice.go:35]: i will leave this for now, as the command itself is just: `storeservice`

**Pull request merged**